samples/bootloader/overlay-minimal-size.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# To reduce size, turn off debug configs, and enable minimal partition size
```

Comment:
What debug configs does this refer to?

Reply:
log?

Reply:
Ah, it looked like an instruction to the user :P. Maybe rephrase it to not be imperative?

Reply:
Haha, yes!